science/marx/Portfile

```diff
@@ -22,17 +22,14 @@ license             GPL-2
 homepage            http://space.mit.edu/cxc/marx/
 master_sites        ftp://space.mit.edu/pub/cxc/marx/v5.3
 
-checksums           rmd160  658fd94bb01afe91ba843a03f56e1ca1a9993faf \
-                    sha256  24aa3734740336f05fef6eb52e1e543965e695e88e148345423f547bcce98b61
+checksums           rmd160  c117e0d48256b08a59e89c9aac61f2cd84a050ce \
+                    sha256  60b8c2a17f3e37910eef48deed4e8d3fd7b51b21748c7e930048298a011f168f
```

Comment:
Please also add a `size` field to the `checksums` section. `port -v checksum` will generate an appropriate stanza for you.